libcaf_test/caf/test/outline.hpp

```diff
@@ -11,7 +11,7 @@
 
 namespace caf::test {
 
-class CAF_TEST_EXPORT outline : public runnable {
+class CAF_TEST_EXPORT outline_setter : public runnable {
```

Comment:
This is not a setter. This is the object that will actually run the tests (examples).

I'd prefer a name like `runnable_with_examples` to have a better description of what it actually is, plus please move this to its own file.

Reply:
Nice suggestion, will do.